go/ql/lib/semmle/go/frameworks/K8sIoApimachineryPkgRuntime.qll

```diff
@@ -192,59 +93,9 @@ module K8sIoApimachineryPkgRuntime {
       // The format is not fixed. It depends on the parameter 0.
       none()
     }
-
-    override predicate hasTaintFlow(DataFlow::FunctionInput inp, DataFlow::FunctionOutput outp) {
-      inp = this.getAnInput() and outp = this.getOutput()
-    }
-  }
-
-  private class NewEncodable extends TaintTracking::FunctionModel {
-    NewEncodable() { this.hasQualifiedName(packagePath(), ["NewEncodable", "NewEncodableList"]) }
-
-    override predicate hasTaintFlow(DataFlow::FunctionInput inp, DataFlow::FunctionOutput outp) {
-      inp.isParameter(1) and outp.isResult()
-    }
   }
 
-  private class UseOrCreateObject extends TaintTracking::FunctionModel {
-    UseOrCreateObject() { this.hasQualifiedName(packagePath(), "UseOrCreateObject") }
-
-    override predicate hasTaintFlow(DataFlow::FunctionInput inp, DataFlow::FunctionOutput outp) {
-      inp.isParameter(3) and outp.isResult(0)
-    }
-  }
-
-  private class ObjectConvertorConvert extends TaintTracking::FunctionModel, Method {
-    ObjectConvertorConvert() { this.implements(packagePath(), "ObjectConvertor", "Convert") }
-
-    override predicate hasTaintFlow(DataFlow::FunctionInput inp, DataFlow::FunctionOutput outp) {
-      inp.isParameter(0) and outp.isParameter(1)
-    }
-  }
-
-  private class ObjectConvertorConvertToVersion extends TaintTracking::FunctionModel, Method {
-    ObjectConvertorConvertToVersion() {
-      this.implements(packagePath(), "ObjectConvertor", "ConvertToVersion")
-    }
-
-    override predicate hasTaintFlow(DataFlow::FunctionInput inp, DataFlow::FunctionOutput outp) {
-      inp.isParameter(0) and outp.isResult(0)
-    }
-  }
```

Comment:
The model for this is missing. (At first glance you might mistake it for `ObjectVersioner.ConvertToVersion`, which is modelled.)

Reply:
I'm satisfied that https://pkg.go.dev/k8s.io/apimachinery/pkg/runtime#ObjectConvertor is a superinterface of https://pkg.go.dev/k8s.io/apimachinery/pkg/runtime#ObjectVersioner and they are defined in the same package, so I don't think we have a problem there